test/integration/api/odata.js

```diff
@@ -1251,6 +1315,36 @@ describe('api: /forms/:id.svc', () => {
             });
           }))));
 
+    it.skip('should limit subtable results', testService(async (service) => {
+      const asAlice = await withSubmissions(service, identity);
+
+      await asAlice.get('/v1/projects/1/forms/withrepeat.svc/Submissions.children.child?$top=10')
+        .expect(200)
+        .then(({ body }) => {
+          const tokenData = {
+            instanceId: body.value[body.value.length - 1].instanceId,
```

Comment:
The strategy of using `instanceId` + `__id` makes sense to me. :+1: How long will this token be? Should we try to compress the data like we do with app user QR codes? (See `CollectQr` in Frontend.)